crates/wysiwyg/src/dom/range.rs

```diff
@@ -72,6 +72,17 @@ impl DomLocation {
         }
     }
 
+    pub fn with_new_handle(&self, handle: DomHandle) -> Self {
```

Comment:
It's probably not a great name, but I couldn't come up with something better.

Reply:
Seems fine to me.